lib/src/constants/constant_text.dart

```diff
@@ -55,3 +55,21 @@ class ForgotPasswordText {
   static const String sendButton = "Send";
   static const String tabletLabel = "Start from Scratch";
 }
+
+class UserProfileText {
+  static const String title = "My Kitchen";
+  static const String icon = "assets/images/user_profile_icon/Settings.png";
+  static const String dot = "assets/images/user_profile_icon/dot.png";
+  static const String edit = "assets/images/user_profile_icon/Edit.png";
+  static const String nav1 = "assets/images/user_profile_icon/nav1.png";
```

Comment:
should we name the variable more clearly ? like recipeFeedIcon, userProfileFeedIcon,...

Reply:
Thanks you, i already fixed name